var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -41,11 +41,13 @@ class Neuron(Package):
     url      = "http://www.neuron.yale.edu/ftp/neuron/versions/v7.5/nrn-7.5.tar.gz"
     git      = "https://github.com/nrnhines/nrn.git"
 
+    version('2018-10', git=git, commit='b3097b7', preferred=True)
+    version('2018-09', git=git, commit='9f36b13')
+    version('7.6.2',   git=git, tag='7.6.2')
     version('7.5', 'fb72c841374dfacbb6c2168ff57bfae9')
     version('7.4', '2c0bbee8a9e55d60fa26336f4ab7acbf')
     version('7.3', '993e539cb8bf102ca52e9fefd644ab61')
     version('7.2', '5486709b6366add932e3a6d141c4f7ad')
-    version('develop', git=git, commit='9f36b132ce1b5e4', preferred=True)
```

Comment:
@ferdonline : we don't want to remove develop because we want an option to build latest neuron. I would keep:

```
version('develop', git=git)
```